contracts/enforcer/HolographGeneric.sol

```diff
@@ -191,6 +191,20 @@ contract HolographGeneric is Admin, Owner, Initializable, HolographGenericInterf
     destination.transfer(address(this).balance);
   }
 
+  function sourceExternalCall(address targetContract, bytes calldata targetPayload) external payable onlySource {
```

Comment:
What is this new function for?

Reply:
It was bleeding over from `feature/cross-chain-multisig` branch. I have removed it to not have overlap.

Reply:
ah okay